src/commentPopup.js

```diff
@@ -0,0 +1,70 @@
+import { addComment, getComments, countComments } from './comment.js';
+
+const commentPopup = async (popupComment, show, num) => {
+  popupComment.innerHTML += `
+    <div class="popup">
+    <span class="close-btn">X</span>
+        <div class="popup-content">
+            
+            <img src="${show.image}" class="image"/>
+            <h1>${show.title}</h1>
+            <div class="popup-detail">
+            </div>
+            <h4 class='count-display'>Comments ${num} </h4>
+            <div class='comments-div' id="com-div-${show.id}">
+
+            </div>
+            <h4>Add a comment</h4>
+            <input type="text" placeholder="Your name" name="name" class="comment-input"/>
+            <input type="textarea" placeholder="Your insights" name="comment" class="comment-input"/>
+            <button class="comment-btn add-comment" id="add-comment-${show.id}">Comment</button>
+        </div>
+        </div>
+    `;
+  document.body.appendChild(popupComment);
+};
+
```

Comment:
- [ ] You have done a great job here. However, when a user adds a comment, it requires the app to be refreshed for a comment to be seen together with the comments counter. Kindly fix this issue so that it works like the likes counter.

![comments](https://user-images.githubusercontent.com/53124548/218266698-a7385fe7-8f23-4a58-ad93-96059647ad1e.png)